src/Documentation/Advanced-Concepts/Activation-Garbage-Collection.md

```diff
@@ -132,6 +132,8 @@ You can also instruct the runtime to deactivate the grain next time it becomes i
 ``` csharp
   protected void DeactivateOnIdle()
 ```
+An activation is considered idle if it is not processing any message at the moment
+and has an empty inbox (i.e. there are no requests waiting to be processed by it).
```

Comment:
This is not quite correct - an activation will get deactivated after completing the request it is processing even if it has more requests enqueued (they will get forwarded to a new activation).

Also, terminology wise, we never talk about inboxes. Using a term like that here would be confusing I'm almost sure.